index.html

```diff
@@ -133,24 +136,41 @@ <h2>Dependencies</h2>
           <p>The following terms are defined in the Fetch specification: [[!FETCH]]</p>
           <ul>
             <li><dfn data-cite="!FETCH#concept-request-client">client</dfn></li>
+            <li><dfn data-cite="!FETCH#cors-preflight-request">CORS-preflight request</dfn></li>
+            <li><dfn data-cite="!FETCH#http-network-fetch">HTTP-network fetch</dfn></li>
+            <li><dfn data-cite="!FETCH#http-network-or-cache-fetch">HTTP-network-or-cache fetch</dfn></li>
+            <li><dfn data-cite="!FETCH#redirect-status" data-lt="redirects">redirect status</dfn></li>
+          </ul>
+        </dd>
+        <dt>HSTS</dt>
+        <dd>
+          <p>The following terms are defined in the HSTS specification: [[!RFC6797]]</p>
+          <ul>
+            <li><dfn data-cite="!RFC6797#section-8.2">superdomain match</dfn></li>
           </ul>
         </dd>
         <dt>HTML</dt>
         <dd>
           <p>The following terms are defined in the HTML specification: [[!HTML]]</p>
           <ul>
+            <li><dfn data-cite="!HTML#navigate" data-lt="navigation">navigate</dfn></li>
             <li><dfn data-cite="!HTML#navigator.online"><code>navigator.onLine</code></dfn></li>
-            <li>resource <dfn data-cite="!HTML#origin">origin</dfn></li>
+            <li>resource <dfn data-cite="!HTML#origin" data-lt="origins">origin</dfn></li>
           </ul>
         </dd>
         <dt>HTTP</dt>
         <dd>
-          <p>The following terms are defined in the HTTP specification: [[!RFC7231]]</p>
+          <p>The following terms are defined in the HTTP specification: [[!RFC7230]], [[!RFC7231]], [[RFC7234]]</p>

```

Comment:
Any particular reason for non-normative ref to 7234?

Reply:
Ah, good catch!  The text below that talks about cached requests not being "network requests" used to be in a note paragraph, which I think would've made any references non-normative?  But I made that paragraph normative, so the reference should be too.

Fixed